index.js

```diff
@@ -10,4 +10,169 @@ let productB = {
     emoji: "🔥",
     revenue: 300,
     commision: 75
-}
\ No newline at end of file
+}
+
+
+/*NOTES:
+1. This is a first draft of the JS algorithm and logic for our app. 
+Let me know if you think this looks clear, or if we should change the work flow of the functions etc.
+2. Some html changes (and CSS?) will be required,
+ and I added a rough list for them in the end of the document.
+3. First part is for algorithm and logic, based on what the app would roughly do. 
+I tried to make it malleable, so if we want to add features, it would be easy to do. 
+But I'm not sure if I was able to take everything into account, so suggestions are welcome :)
+4. Second part is for specifying a few core concepts, and what they are supposed to do in this first draft. 
+ We might want to change some of it for the next draft.
+5. I added some other comments along the way too.
+6. I didn't add the localStorage yet! I've had some issues getting it to work before, 
+so I need to brush up on my knowledge on it first...
+ */
+
+/* ALGORITHM & LOGIC
+
+renderLayoutOne() 
+
+renderLayoutOne(){ /*this would be the "choose your pet"-section 
+
+	render the pop-up-window visible with css.style
+
+	OR
+
```

Comment:
Ahh yes, I haven't started on the React course yet :/

Do you mean we should move the game section into the js file, and render it with innerHtml? Or have it in the html, and hide it initially with CSS? 

My idea here was to have the game section in the html, and render the "choose your pet" with js, to indicate that the game is the main content, but i honestly dont know if that matters 😅

Reply:
I think it'd be easier to render the game in JS, because some elements there are conditional (pet name, pet type) and everything has to be interactive, wheras the 'choose your pet' section can have hardcoded non-changing elements

Reply:
Yeah, sounds reasonable. Lets do that!